kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -804,5 +758,179 @@ def log2_stride_fr(self):
     def stride_fr(self):
         return (2**self.log2_stride_fr)
 
+    _doc_instantiation_shape = {True:
+                                'S = TimeFrequencyScattering(J_time, J_freq, Q, N)',
```

Comment:
I don't necessarily agree here since the parameters are not literals for the docstring, but variables (`J_time, J_freq, Q, N`). As such, it's pretty clear what each argument stands for. That being said, I've made the change, so the instantiation now looks like

```
S = TimeFrequencyScattering(J=J_time, J_fr=J_freq, Q=Q, shape=N)
```

(A bit verbose, if you ask me, but the users should get the drift.)